tests/e2e/api/controllers/users.js

```diff
@@ -844,6 +844,102 @@ describe('Users API', () => {
           .then(() => expectPasswordLoginToWork(user));
       });
 
+      it('should create and update many users correctly w/o token_login', async () => {
```

Comment:
Nice! 

I think we should have a similar test that creates users with token_login enabled, and one where some users fail to be created, so we can assert on the response we get e2e. 
Thanks!